cmd/rancher-desktop-guestagent/main.go

```diff
@@ -98,59 +97,65 @@ func main() {
 		cancel()
 	}()
 
-	tcpTracker := tcplistener.NewListenerTracker()
+	if !*enableContainerd &&
+		!*enableDocker &&
+		!*enableIptables {
+		log.Fatal("requires either -docker, -containerd or -iptables enabled.")
+	}
 
-	wslAddr, err := getWSLAddr(wslInfName)
-	if err != nil {
-		log.Fatalf("failure getting WSL IP addresses: %v", err)
+	if *enableContainerd &&
+		*enableDocker &&
+		*enableIptables {
+		log.Fatal("requires either -docker, -containerd or -iptables, not all.")
```

Comment:
That means I can still start with `--docker --containerd` (because `--iptables` is not set). Is that correct?
It might be good to include a table as a comment in this PR to describe what the desired state is.